### PR TITLE
feat: stop populating first class event fields

### PR DESCRIPTION
**Pull request by kotharironak**
## Description
This is in the context of
-  https://github.com/hypertrace/hypertrace/issues/244
- https://github.com/hypertrace/hypertrace/issues/245

### Testing
- Fixed the migration test after the change.

### Note
- all the dependent (consumer) changes to make this possible have been merged.
- we will also be removing these fields from the [data model](https://github.com/hypertrace/data-model/blob/main/data-model/src/main/avro/Event.avdl#L59) as a follow-up.

### CODE REVIEW

**Review by rish691 — COMMENTED** [review]
Build is failing, tests have to be updated accordingly

**Comment by kotharironak** [comment]
> Build is failing, tests have to be updated accordingly

But, do we need this test now? It's trying to covert to RawSpna. I can use a field generator.

File: hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestHttp.java [hunk 1]
```diff
@@ -199,45 +208,50 @@ public void testInvalidRequestPath() throws Exception {
...
+      throws Exception {
 
     Span span = createSpanFromTags(tagsMap);
     RawSpan rawSpan = normalizer.convert("tenant-key", span);
 
-    assertEquals(
-        rawSpan.getEvent().getHttp().getRequest().getUserAgent(),
-        HttpSemanticConventionUtils.getHttpUserAgent(rawSpan.getEvent()).get());
+    // now, we are not populating first class fields. So, it should be null.
+    assertNull(rawSpan.getEvent().getHttp());
```
Comment (sarthak77) [inline hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestHttp.java:222]: Isn't this repetitive/irrelevant in every test ? Maybe we can have another test just checking nullity of first class fields?

**Comment by sarthak77** [comment]
We need to merge the changes from main too

File: hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestHttp.java (hunk 1 above)
Comment (kotharironak) [reply hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestHttp.java:222]: I kept it after - `RawSpan rawSpan = normalizer.convert("tenant-key", span);` - convert call.

**Review by rish691 — APPROVED** [review]

**Pull request merged**